internal/resources/resource_volume.go

```diff
@@ -326,6 +371,26 @@ func resourceMetalVolumeRead(d *schema.ResourceData, meta interface{}) (err erro
 		return fmt.Errorf("set Size: %v", err)
 	}
 
+	if err = d.Set(vSizeInUse, math.Round(float64(volume.CapacityUsed)/KiBToGBConversion)); err != nil {
+		return fmt.Errorf("resourceMetalVolumeRead: set %s : %v", vSizeInUse, err)
```

Comment:
I'd remove "resourceMetalVolumeRead" from the error messages to be more secure.

Reply:
As this is a public repo, I decided to put more context to the error on where it is coming from.
Anyway, I have removed the method names to align with other implementations. 
Thanks.